crates/common/types/transaction.rs

```diff
@@ -1580,4 +1681,63 @@ mod tests {
             serde_json::from_str(generic_transaction).unwrap()
         )
     }
+
+    #[test]
+    fn deserialize_eip4844_transaction() {
+        let eip4844_transaction = r#"{
+            "chainId":"0x01",
+            "nonce":"0x02",
+            "maxPriorityFeePerGas":"0x01",
+            "maxFeePerGas":"0x01",
+            "gas":"0x5208",
+            "to":"0x6177843db3138ae69679A54b95cf345ED759450d",
+            "value":"0x01",
+            "input":"0x",
+            "accessList": [
+                {
+                    "address": "0x000f3df6d732807ef1319fb7b8bb8522d0beac02",
+                    "storageKeys": [
+                        "0x000000000000000000000000000000000000000000000000000000000000000c",
+                        "0x000000000000000000000000000000000000000000000000000000000000200b"
+                    ]
+                }
+            ],
+            "maxFeePerBlobGas":"0x03",
+            "blobVersionedHashes": [
+                    "0x0000000000000000000000000000000000000000000000000000000000000001",
+                    "0x0000000000000000000000000000000000000000000000000000000000000002"
+            ],
+            "yParity":"0",
```

Comment:
We can use the number in hex to fully test the behavior:

```suggestion
            "yParity":"0x0",
```